envs/env-py312.yml

```diff
@@ -206,7 +209,7 @@ dependencies:
     # - gladier-tools
     - h5json
     - mimesis
-    - ophyd-async[ca,pva]
+    - ophyd-async[ca,pva] >=0.9.0
```

Comment:
For `ophyd-async` there is an alpha release: `0.10.0a2` - any chance we could use that (or I guess release a new env when `0.10.0` is officially released?

Reply:
Sure, let's add the latest version v0.10.0a2.